packages/odepack/odepack.0.7/opam

```diff
@@ -0,0 +1,33 @@
+opam-version: "2.0"
+maintainer: "Christophe Troestler <Christophe.Troestler@umons.ac.be>"
+authors: [ "Christophe Troestler <Christophe.Troestler@umons.ac.be>" ]
+license: "LGPL-3.0 with OCaml linking exception"
+homepage: "https://github.com/Chris00/ocaml-odepack"
+dev-repo: "git+https://github.com/Chris00/ocaml-odepack.git"
+bug-reports: "https://github.com/Chris00/ocaml-odepack/issues"
+doc: "https://chris00.github.io/ocaml-odepack/doc/"
+tags: [ "ODE" "scientific-computing"  ]
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "build" "@doc"] {with-doc}
+]
+run-test: [
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
```

Comment:
this doesn't work. It's either a command in `run-test` without `{with-test}` or a command in `build` with `{with-test}`. The combination of the two will just do nothing

Reply:
it's fixed now. Could you return the fix upstream?

Reply:
[Done](https://github.com/Chris00/ocaml-odepack/commit/32e92b57093333db4acb2d8b6a8b8eac123df63c).  Thanks.